README.md

```diff
@@ -1,5 +1,7 @@
 # FFI::Aspell
 
+[![Build Status](https://travis-ci.org/YorickPeterse/ffi-aspell.svg?branch=master)](https://travis-ci.org/YorickPeterse/ffi-aspell)
+
```

Comment:
Could you remove this? I don't really see the point of adding this to the README. End users are not likely to give a damn about what the CI status is of the master branch.